wandering-warriors/abacus.kv

```diff
@@ -0,0 +1,2 @@
+<Abacus>:
```

Comment:
I'm not sure what this file is for and why it just contains an Abacus widget, and why the Abacus widget has a canvas element

Reply:
kv file for detailing widgets. https://kivy.org/doc/stable/guide/lang.html .
Just built in the syntax for loading a separate file so we don't have to keep massive kv strings in line with the python code

Reply:
canvas is just a blank thing to load. agian, just a starting place.